sources/subs/MessageIndex.subs.php

```diff
@@ -168,7 +168,6 @@ function topicsParticipation($id_member, $topic_ids)
 		FROM {db_prefix}messages
 		WHERE id_topic IN ({array_int:topic_list})
 			AND id_member = {int:current_member}
-		GROUP BY id_topic
```

Comment:
Actually, I think you need it, or in alternative you need a DISTINCT because messages holds a lot of id_topic per each message and without the aggregation you potentially get an awful amount of results that will be looped both for fetching and for processing later on. :wink:

Reply:
I'll reverse that change, then.